packages/browser-runtime-electron/README.md

```diff
@@ -17,7 +17,7 @@ const runtime = new ElectronRuntime(serviceProvider);
 import { Shell, ElectronRuntime } from 'mongosh-browser-repl';
 
 const runtime = new ElectronRuntime(
-  CompassServiceProvider.fromDataService(dataService)
+  await CompassServiceProvider.connect(...)
```

Comment:
Do we want maybe to have some simple example of what can be used as the function arg instead of `...?

Reply:
I'm ... leaning towards no? This example isn't meant to be even close to runnable, it's intended to demonstrate how the different pieces of architecture (ElectronRuntime, ServiceProvider, browser-repl) work together.